worker/tracing/manifold_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package tracing
```

Comment:
I'd consider just `trace`. We have lots of {verb}er; this is {verb}ing...

Reply:
Sure.